cmd/config.go

```diff
@@ -38,12 +38,12 @@ var applyCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(configCmd)
-	configCmd.AddCommand(generateCmd)
-	configCmd.AddCommand(applyCmd)
+	configCmd.AddCommand(configGenerateCmd)
+	configCmd.AddCommand(configApplyCmd)
 	//
-	generateCmd.Flags().BoolVarP(&infra, "infra", "", false, "generate infra config")
+	configGenerateCmd.Flags().BoolVarP(&infra, "infra", "", false, "generate infra config")
 	generateCmd.Flags().BoolVarP(&workload, "workload", "", false, "generate workloads config")
```

Comment:
this belongs to configGenerateCmd

Reply:
fixed, we can consider removing the config cmd as well